src/middlewared/middlewared/plugins/enclosure_/map.py

```diff
@@ -96,7 +96,8 @@
             MappingSlot(0, 15, False),
         ]),
     ]),
-    ProductMapping(re.compile(r"TRUENAS-R20$"), [
+    #R20 and R20B share chassis and mapping
```

Comment:
flake8 funness 😄 comments need to start like `# ` (a space after the pound-sign)